guides/source/upgrading_ruby_on_rails.md

```diff
@@ -358,13 +358,18 @@ puts Rails.logger.broadcasts #=> [MyLogger]
 Active Record Encryption now uses SHA-256 as its hash digest algorithm. If you have data encrypted with previous Rails
 versions, there are two scenarios to consider:
 
-1. If you have `config.active_support.key_generator_hash_digest_class` configured as SHA-1 (the default
-   before Rails 7.0), you need to configure SHA-1 for Active Record Encryption too:
+1. If you have `config.active_support.key_generator_hash_digest_class` configured as SHA-1 (the default before Rails 7.0) or have not explicitly set a configuration, you need to configure SHA-1 for Active Record Encryption too:
```

Comment:
> or have not explicitly set a configuration

I'm not sure this adds any more clarity to the sentence or just adds more words.

Personally I am partial to "If X then Y", especially when inside ordered lists. This is closer to "if X (and Y) or Z then...", but just a really small knit.

Appreciate when people go back to update the docs after going through something as challenging as this process can be.

Reply:
I think when making this change initially I read this as referring to `config.active_record.encryption.hash_digest_class`. I don't know that this part of the change was needed.